src/+groundwater/readGroundwaterSettings.m

```diff
@@ -94,32 +86,32 @@ soilThick           cumulative soil layers thickness (from top to bottom)
         end
     end
 
-    indxBotmLayer_R = indxBotmLayer_R; % index of bottom layer that contains current headBotmLayer
+    % indxBotmLayer_R: index of bottom layer that contains current headBotmLayer
```

Comment:
I think we can remove this line at all. It was my mistake that I didn't remove it